frontend/src/config/utils.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import path from 'path';
+
+import type { BeamerConfig } from '@/types/config';
+
+import { ChainMetadata } from './chains/chain';
+import { DeploymentInfo } from './deployment';
+import { TokenMetadata } from './tokens/token';
+import type { Environment } from './types';
+
```

Comment:
I think it will be a good improvement for the future as the codebase grows.
So, in order to have less work in the future when we turn on the `explicit-function-return-type` in the linter, i will adjust the code to explicitly type the returns.
Good idea!